arangod/Statistics/Descriptions.cpp

```diff
@@ -361,11 +361,25 @@ void stats::Descriptions::serverStatistics(velocypack::Builder& b) const {
   if (dealer->isEnabled()) {
     b.add("v8Context", VPackValue(VPackValueType::Object, true));
     auto v8Counters = dealer->getCurrentContextNumbers();
+    auto memoryStatistics = dealer->getCurrentMemoryNumbers();
     b.add("available", VPackValue(v8Counters.available));
     b.add("busy", VPackValue(v8Counters.busy));
     b.add("dirty", VPackValue(v8Counters.dirty));
     b.add("free", VPackValue(v8Counters.free));
     b.add("max", VPackValue(v8Counters.max));
+    {
+      b.add("memory", VPackValue(VPackValueType::Array));
+      for (auto memStatistic : memoryStatistics) {
+        b.add(VPackValue(VPackValueType::Object));
+        b.add("contextId", VPackValue(static_cast<int32_t>(memStatistic.id)));
```

Comment:
the cast is unnecessary here and should be removed
```suggestion
        b.add("contextId", VPackValue(memStatistic.id));
```